notebooks/gym_pytorch.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Awesome, thank you!  We may mangle this a bit to fit it into a larger notebook reorg that's coming in a bit, but this is already a really helpful example.

Reply:
By the way, I'll change the branch name in the colab sticker to point to main if that's alright with you.

Reply:
Makes sense, thanks.